<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QWidget" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Смена пароля</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/source/konoha.ico</normaloff>:/source/konoha.ico</iconset>
  </property>
  <widget class="QPushButton" name="ChangeButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="oldBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Старые данные</string>
   </property>
   <widget class="QLineEdit" name="oldLoginEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="oldPasswordEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="oldLoginLabel">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий логин:</string>
    </property>
   </widget>
   <widget class="QLabel" name="oldPasswordLabel">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий пароль:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="newBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Новые данные</string>
   </property>
   <widget class="QLineEdit" name="newLoginEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="newPasswordEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="newLoginLabel">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Новый логин:</string>
    </property>
   </widget>
   <widget class="QLabel" name="newPasswordLabel">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новый пароль:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="CheckLabel">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>170</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Старые данные неверны</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
